e2e/specs/st_radio.spec.js

```diff
@@ -49,9 +49,9 @@ describe("st.radio", () => {
       .should(
         "have.text",
         "value 1: female" +
-        "value 2: female" +
-        "value 3: None" +
-        "value 4: female"
+          "value 2: female" +
+          "value 3: None" +
+          "value 4: female"
```

Comment:
Yes, this is changes somehow happen during merge with the latest `develop`

Reply:
Hm, looks like some recent changes edited the files in the e2e/specs folder so that prettier wants to auto-format them.

I made a separate PR (https://github.com/streamlit/streamlit/pull/4241) so that they don't appear in this diff